docs/Concise-Guide-for-Evaluating-Open-Source-Software.md

```diff
@@ -37,7 +37,10 @@ As a software developer, before using open source software (OSS) dependencies or
 6. **Are there instructions on how to report vulnerabilities?** See the [Guide to implementing a coordinated vulnerability disclosure process for open source projects](https://github.com/ossf/oss-vulnerability-guide/blob/main/maintainer-guide.md#guide-to-implementing-a-coordinated-vulnerability-disclosure-process-for-open-source-projects) for guidance to OSS projects.
 7. **Does it have significant use?** Software with many users or large users may be inappropriate for your use. However, widely used software is more likely to offer useful information on how to use it securely, and more people will care about its security. Check if a similar name is more popular - that could indicate a typosquatting attack.
 8. **What is the software’s license?** Licenses are technically not security, but licenses can have a significant impact on security and sustainability. Ensure every component has a license, that it’s a widely-used [OSI license](https://opensource.org/licenses) if it’s OSS, and that it’s consistent with your intended use. Projects that won’t provide clear license information are less likely to follow other good practices that lead to secure software.
-9. **What are the results of code evaluation?** Even a brief review of software (by you, someone you hire, or someone else), along with recent changes to it, can give you some insight. Here are things to consider:
+9. **What happens on a tentative addition?** Try adding the dependency as a test, preferably in an isolated environment (in case it's malicious).
+   1. Does it exhibit malicious behaviour, e.g., does it attempt to exfiltrate sensitive data?
+   2. What new indirect dependencies does it add? Is they unexpectedly many or unnecessary? Ask components to reduce runtime dependencies (e.g., by having them differentiate between runtime, development time, and test time dependencies).
```

Comment:
"Is they unexpectedly many or unnecessary?" doesn't look right and I'm not sure enough about what you meant to make a suggestion.

Reply:
Basically, if you suddenly double the number of dependencies, find yourself adding compilers & such in operations, etc. Not sure how to write that text though.

Reply:
I'll rewrite that text, hopefully it will become clearer.